README.md

```diff
@@ -0,0 +1,37 @@
+# Hathor Wallet Service -- Sync Daemon
+
+## State Machine
+
+The state machine diagram can be visualized at https://gist.github.com/andreabadesso/7299c0ed0ce189bc121a06dce1e11638
```

Comment:
Nice to have this gist but I think we should copy the latest image here (or render automatically if it is possible).

Reply:
Updated the URL to render automatically from my gist, here is the URL I added to the description:
https://xstate.js.org/viz/?gist=7299c0ed0ce189bc121a06dce1e11638

Commit hash is cb1a5b36aaa63d354997cb646a0f5ff72b8b49ce

Reply:
Great to have the most updated link. I also think we should have a copy in the repository (in case the link stops working).